gravitee-cockpit-connectors-core/src/main/java/io/gravitee/cockpit/connectors/core/services/AlertManagerService.java

```diff
@@ -72,6 +86,26 @@ private Completable registerCockpitTriggers() {
                 }
             )
             .doOnSuccess(triggers -> triggers.forEach(triggerProvider::register))
+            .map(triggers -> (triggersId = triggers.stream().map(Trigger::getId).collect(Collectors.toSet())))
             .ignoreElement();
     }
+
+    private void doOnCommand(Command command) {
+        if (command instanceof io.gravitee.alert.api.trigger.command.AlertNotificationCommand) {
+            io.gravitee.alert.api.trigger.command.AlertNotificationCommand alertNotificationCommand = (io.gravitee.alert.api.trigger.command.AlertNotificationCommand) command;
+            if (triggersId.contains(alertNotificationCommand.getTrigger())) {
+                cockpitConnector.sendCommand(convert(alertNotificationCommand));
```

Comment:
Shouldn't we need a `subscribe` to trigger the call? 🤔

Reply:
You re right !